### PR TITLE
[imp] install lxml via pip

### PR DESCRIPTION
**Pull request by simahawk**
`pip install lxml` is working today ;)

/cc @nhomar

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,3 +2,5 @@ simplejson
 pyserial
 pyyaml
 coveralls
+lxml
+click
```
Comment (moylop260) [inline requirements.txt]: Click?

Comment (simahawk) [inline requirements.txt]: click is related to #356 
Unfortunately I didn't used to separated branches BUT @nhomar knows how to split these commits on merge, and he told me it was fine for such a small change :)

Comment (moylop260) [inline requirements.txt]: We have installed click from install_nightly script when is required.
Then we dont need it here.

IMHO here should be just odoo requirements

**Comment by simahawk** [comment]
​the issue is that if you use this on your local machine​ it won't work :)

**Comment by moylop260** [comment]
Using the same docker image used by travis to compare the time:
 - `docker run -it --rm quay.io/travisci/travis-python:latest /bin/bash -c "time apt-get install python-lxml"`
  - 9 seconds

 - `docker run -it --rm quay.io/travisci/travis-python:latest /bin/bash -c "time pip install lxml"`
  - 52 seconds

screen shot 2016-10-04 at 2 12 23 pm

**Comment by pedrobaeza** [comment]
So it seems to not be the same speed. I close the PR then.

**Pull request closed without merging**